app/main.py

```diff
@@ -1 +1,42 @@
-# write your code here
+from typing import Callable
+
+
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def take_damage(self, amount: int) -> Callable:
```

Comment:
Change your return type annotation, you do not return `Callable` here and in the `die` and `hide` methods as well